app/dashboard/sync/binance.py

```diff
@@ -30,10 +30,6 @@ def get_binance_txn_status(fulfillment):
             'params': [ txnid ]
         }
 
-        headers = {
-            'Host': 'gitcoin.co'
```

Comment:
Was removing this from Binance part of the Casper integration or just cleanup?

Reply:
just cleanup.. it's unused